docker/proxy/Dockerfile

```diff
@@ -1,27 +1,25 @@
-FROM --platform=${BUILDPLATFORM} golang:1.20 AS builder
+FROM --platform=${BUILDPLATFORM} golang:1.20-alpine AS builder
 ARG TARGETARCH
 ARG TARGETOS
 
 WORKDIR /workdir
 
 COPY . .
-RUN apt-get -y update && \
-    apt-get -y install protobuf-compiler && \
+RUN apk add --no-cache bash build-base protobuf && \
```

Comment:
Please also install `git` here. This is not included in the Go `-alpine` image, but is used to determine the version of the signaling proxy (would be `unknown` now).

Reply:
done